awx/main/tasks.py

```diff
@@ -1997,10 +2001,7 @@ class RunProjectUpdate(BaseTask):
 
     @property
     def proot_show_paths(self):
-        show_paths = [settings.PROJECTS_ROOT]
-        if self.job_private_data_dir:
-            show_paths.append(self.job_private_data_dir)
-        return show_paths
+        return [settings.PROJECTS_ROOT]
```

Comment:
?

Reply:
? as in, "why did remove the job dir here" or ? as in, "why did use `@property` when I could have make it an actual property?"

For the latter question, the answer is CTiT, I don't want it to be looking up the setting on load of the module.

The former question is much bigger. I removed the `synchronize` module from the playbook. By doing this, the duties of the project update playbook become much more static, because given a certain project, it always downloads to the same "stage" folder for the roles/collections. Thinking there relates to https://github.com/ansible/awx/issues/7625, I want to have a project update interface that works for that, while also working for caching roles / collections. The project cache folder is already a subfolder of the PROJECTS_ROOT folder, so it no longer needs to be added here.

I'm going to be writing docs on this. The overall function will be much easier to explain with these changes.